sviluppo/review.rst

```diff
@@ -57,57 +62,88 @@ Revisione tecnica
 
 Per fare una revisione del codice è necessario un buon livello di conoscenza di Python e Odoo.
 
-Accedere al tab **Files changed**
+Accedere alla scheda **Files changed**.
 
-.. image:: ./immagini/github_pr_review_changes.png
+.. image:: ./immagini/github_pr_files_changed.png
+   :align: center
 
-Vengono visualizzate le parti di codice che sono state modificate
+Vengono visualizzate le parti di codice che sono state modificate.
 
-.. image:: ./immagini/github_pr_review_r2.png
+.. image:: ./immagini/github_pr_review_r1.png
+   :align: center
 
-Verificare la correttezza dei cambiamenti proposti e la loro coerenza con le `linee guida OCA <https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst>`_ e le :doc:`linee guida Italiane <guidelines>`.
+Verificare la correttezza dei cambiamenti proposti e la loro coerenza con le `linee guida OCA <https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst>`_ e le :doc:`linee guida italiane <guidelines>`.
```

Comment:
![image](https://github.com/odoo-italia/documentazione/assets/3512779/74b74e5a-397a-456a-96cf-5cc7f225fc16)
Il link evidenziato mi sembra non corretto: ora punta a https://github.com/primes2h/documentazione/blob/REF_functional_review/sviluppo/review.rst#id1

Reply:
> ![image](https://private-user-images.githubusercontent.com/3512779/343669788-74b74e5a-397a-456a-96cf-5cc7f225fc16.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTk0NzU1MjMsIm5iZiI6MTcxOTQ3NTIyMywicGF0aCI6Ii8zNTEyNzc5LzM0MzY2OTc4OC03NGI3NGU1YS0zOTdhLTQ1NmEtOTZjZi01Y2M3ZjIyNWZjMTYucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDYyNyUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA2MjdUMDgwMDIzWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9ZDk2ZWRjMzc1ZjY5MGU3MDQwMjQ0OGFlZmJmZmM3NjAyY2M4YzM0ZDA4N2FmMzU5ZjU2YjU3MmYyMzFiMDc2ZCZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.iltncQsMlIZqZLKGQ9QH4ukCa5qciZOlvrja4mOzLxs) Il link evidenziato mi sembra non corretto: ora punta a https://github.com/primes2h/documentazione/blob/REF_functional_review/sviluppo/review.rst#id1

Quel link viene generato dinamicamente in fase di compilazione. A te compare così perché stai compilando dal branch della PR direi.

Una volta fatto il merge e ricompilata la documentazione sul server dovrebbe puntare al link corretto, vedi ad esempio la pagina attuale:
https://www.odoo-italia.org/documentazione/14.0/sviluppo/review.html

Reply:
Perfetto, grazie 1K!

Reply:
:+1: